<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>498</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>600</width>
      <height>481</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="piedra">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>30</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>piedra</string>
    </property>
    <property name="iconSize">
     <size>
      <width>5</width>
      <height>5</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="papel">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>90</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>papel</string>
    </property>
    <property name="iconSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="tijera">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>150</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>tijera</string>
    </property>
    <property name="iconSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="ingresarJugador">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>220</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>ingresar jugador</string>
    </property>
   </widget>
   <widget class="QPushButton" name="inicio">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>270</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>inicio</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="tiempoLabel">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>320</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
    <property name="smallDecimalPoint">
     <bool>true</bool>
    </property>
    <property name="digitCount">
     <number>5</number>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Filled</enum>
    </property>
    <property name="value" stdset="0">
     <double>0.000000000000000</double>
    </property>
    <property name="intValue" stdset="0">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLCDNumber" name="puntajeLabel">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>360</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="puntajePiedra">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>50</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="puntajePapel">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>110</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="puntajeTijeras">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>170</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>graphicsView</sender>
   <signal>rubberBandChanged(QRect,QPointF,QPointF)</signal>
   <receiver>tiempoLabel</receiver>
   <slot>setBinMode()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>512</x>
     <y>392</y>
    </hint>
    <hint type="destinationlabel">
     <x>513</x>
     <y>456</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
